gzipstream/tests/test_gzipstreamfile.py

```diff
@@ -0,0 +1,23 @@
+from .. import gzipstreamfile
+import unittest
+
+GzipStreamFile = gzipstreamfile.GzipStreamFile
+
+
+class TestSequenceFunctions(unittest.TestCase):
```

Comment:
@tmylk I think moving it into the same subdirectory with the other tests is the easiest way to go. Let me know if this is acceptable for you.